### ISSUE TITLE
About GPU Memory

### ISSUE THREAD

**Issue by jianglh-WHU** [Haian-Jin/LVSM#24]
Hi @Haian-Jin,

Thank you for your excellent work!

I trained the LVSM decoder only on the preprocessed Re10k dataset. However, I noticed that the GPU memory usage is only 24GB. Is this normal?

<img width="605" height="93" alt="Image" src="https://github.com/user-attachments/assets/6a85907e-4ff4-463c-8b25-5f1967b1022b" />

If this is expected, could you explain why you didn’t increase the batch size to more fully utilize the available resources? If not, could you kindly share the reason behind this?

Looking forward to your insights.

**Comment by Haian-Jin** [comment]
What training setting are you using?

**Comment by jianglh-WHU** [comment]
Hi, I just follow the released [decoder_only config](https://github.com/Haian-Jin/LVSM/blob/main/configs/LVSM_scene_decoder_only.yaml):
image_size: 256 / patch_size: 8 / transformer dim: 768 / depth layer: 24 / input views: 2 / output_views: 6

**Comment by Haian-Jin** [comment]
Hi! I used A100 GPUs with 40GB of memory for my experiment. 

While I don’t recall the exact amount of GPU memory used, I remember it wasn’t fully utilized. However, the key point is that the GPU utilization rate was 100% for most time, which implies that increasing the batch size would actually slow down training.

To maximize training speed, I opted to use a smaller batch size distributed across more GPU nodes for faster training. There is a trade-off between the number of GPUs used and training time. You can adjust this based on your personal settings.

**Comment by jianglh-WHU** [comment]
I got it, thank for your reply!
